tests/controller/README.md

```diff
@@ -1 +1,209 @@
+
+###### Iteration 1:
+```shell
+registry resolve projects/demo/locations/global/artifacts/test-manifest
+```
+This will generate and execute the same 30 actions mentioned above. You can see the artifacts generated by the controller by doing a list call
+`registry list projects/demo/locations/global/apis/-/versions/-/specs/-/artifacts/-`
+
+###### Iteration 2:
+Note that  we haven't generated the additional 10 lintstats-spectral artifacts yet. 
+```shell
+registry resolve projects/demo/locations/global/artifacts/test-manifest
+```
+After the first iteration, the dependencies (`$resource.spec/artifacts/lint-spectral`) are generated, and hence in this iteration, the controller will generate and execute 10 actions.
+* 10 `registry compute lintstats ...`
+A quick check with a list call can verify things 
+`registry list projects/demo/locations/global/apis/-/versions/-/specs/-/artifacts/-`
+
+To summarize, in the standalone mode, the controller can bring the registry up-to-date in two iterations.
+
+##### Continuous mode:
+In this mode, the controller is running continuously, making sure that it is always checking the state of the registry in each passing iteration. This can be achieved through a GKE cron job.
+
+###### Step 1: Building the docker image: 
+Build the base image which includes the registry tool:
+```shell
+# Build
+docker build -f containers/registry-tools/Dockerfile . -t gcr.io/$REGISTRY_PROJECT_IDENTIFIER/registry-tools
+
+# Push
+docker push gcr.io/$REGISTRY_PROJECT_IDENTIFIER/registry-tools:latest
+```
+
+Build the linter image from the base image which includes the binaries necessary for the `test-manifest` to work.
+```
+# Build
+envsubst < containers/registry-tools/registry-linters/Dockerfile | docker build -t gcr.io/$REGISTRY_PROJECT_IDENTIFIER/registry-linters -f - .
+
+# Push
+docker push gcr.io/$REGISTRY_PROJECT_IDENTIFIER/registry-linters:latest
+```
+
+###### Step 2: Deploying the job
+Deploy the controller job which will use the `registry-linters` image and run the `registry resolve` command, every 5 mins
+```shell
+export REGISTRY_MANIFEST_ID=projects/demo/locations/global/artifacts/test-manifest
+
+make deploy-controller-job
+```
+
+###### Step 3: Verification
+* Once the job is deployed, you should see that in every iteration it is reading the manifest and calculating the actions. 
+* Since, we already have the artifacts generated fom our previous commands, we need to clean up the previously generated artifacts. 
+`registry delete projects/demo/locations/global/apis/-/versions/-/specs/-/artifacts/-`.
+Once the artifacts are cleaned up, we can see the controller in action.
+* Use list command to check what artifacts are generated by the controller, the behavior should be same as the one. described in the standalone case.
```

Comment:
>the behavior should be same as the one. described

I think this should say "be the same" and the period after "one" seems incorrect.

Reply:
Done.